openssl/src/argon2.rs

```diff
@@ -0,0 +1,147 @@
+use libc::c_void;
```

Comment:
You can get this from std https://doc.rust-lang.org/stable/std/ffi/enum.c_void.html